src/components/VideoPlayerContexts/types.ts

```diff
@@ -27,8 +28,15 @@ type VideoPopoverMenuContext = {
     updatePlaybackSpeed: (speed: PlaybackSpeed) => void;
 };
 
+type FullScreenContext = {
+    isFullscreen: MutableRefObject<boolean>;
+    lockedWindowDimensions: MutableRefObject<WindowDimensions | null>;
```

Comment:
if these 2 properties are of type `MutableRefObject` then do you think it makes sense to add `*Ref` to their names? It seems that very often in this app refs are named like `screenRef` etc.

So in this case `lockedWindowDimensionsRef`? although it does sound weird and the `isFullscreen` prop would sound even weirder.